apidoc/Facebook.yml

```diff
@@ -891,6 +891,46 @@ methods:
     since: "6.1.0"
     platforms: [iphone, ipad]
 
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Dictionary of data received in push notifications callback
+        type: Dictionary
+
+  - name: setPushNotificationsDeviceToken
+    summary: dictionary of data received in the push notification
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Device Token received when registering for push notifications
+        type: String
+
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [android]
```

Comment:
Android is taking 1 more parameter actionID, which iOS probably also should, but I have already tested it without this param...